Formula/owamp.rb

```diff
@@ -17,6 +17,8 @@ class Owamp < Formula
   # Fix to prevent tests hanging under certain circumstances.
   # Provided by Aaron Brown via perfsonar-user mailing list:
   # https://lists.internet2.edu/sympa/arc/perfsonar-user/2014-11/msg00131.html
```

Comment:
Could you see if this was ever submitted upstream and, if not, consider doing so? Thanks!